app/src/pages/inside/logsPage/historyLine/historyLineItem/historyLineItemTooltip/historyLineItemTooltip.jsx

```diff
@@ -71,12 +76,19 @@ const messages = defineMessages({
 });
 
 @connect(
-  (state) => ({
-    activeProject: activeProjectSelector(state),
-    includeAllLaunches: includeAllLaunchesSelector(state),
-  }),
+  (state) => {
+    const currentPage = pageSelector(state);
+    return {
+      activeProject: activeProjectSelector(state),
+      includeAllLaunches:
+        currentPage === PROJECT_LOG_PAGE
+          ? includeAllLaunchesSelector(state)
```

Comment:
Since this logic for controlling `includeAllLaunches` differs depending on where we use this component, it would be better to transfer control to a higher level. What does this mean. We pass this selector as a prop through the parent component. This will help us get rid of additional checks since the parent will be responsible for passing the required method.